library/extractor/src/main/java/com/google/android/exoplayer2/extractor/mkv/MatroskaExtractor.java

```diff
@@ -1269,6 +1269,9 @@ protected void binaryElement(int id, int contentSize, ExtractorInput input) thro
         handleBlockAdditionalData(
             tracks.get(blockTrackNumber), blockAdditionalId, input, contentSize);
         break;
+      case ID_BLOCK_ADD_ID_EXTRA_DATA:
```

Comment:
@steffenmanden I think you missed this part in my original code ;)
Check for:
```
case ID_BLOCK_ADD_ID_EXTRA_DATA:
	currentAdditionMapping.extraData = new byte[contentSize];
	input.readFully(currentAdditionMapping.extraData, 0, contentSize);
	Log.i("yusesope",String.format("ID_BLOCK_ADD_ID_EXTRA_DATA size = %d", currentAdditionMapping.extraData.length));
```
See there was one of my log lines you hate so much!
I need them to debug the code ;)
Then I delete them in one go with a regExp:
`Log.\w\("yusesope".*?;\n`

Reply:
I know we use Logs - usually i keep them locally and dont commit them - if you prefer to have them there its OK :D as long as you promise we will clean up

Reply:
Nessun problema. Farò lo stesso!
Era solo per spiegarti il perchè :)!

Reply:
But now you can see why Github is nice - we can see each others code changes and comment them - and then make a shared codebase :)